### PR TITLE
Add credit card endpoints

### PR DESCRIPTION
**Pull request by averevki**

### CODE REVIEW

**Review by JaurbanRH — DISMISSED** [review]

**Review by jsmolar — CHANGES_REQUESTED** [review]
Please add tests.

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -68,6 +74,11 @@ def master_api(master_url: str, master_token: str,
                                            ssl_verify=ssl_verify)
 
 
+@pytest.fixture(scope='session')
+def stripe_init(stripe_api_key):
```
Comment (jsmolar) [inline tests/integration/conftest.py:78]: use autouse :) https://docs.pytest.org/en/6.2.x/fixture.html

**Comment by averevki** [comment]
@mganisin can you please review this PR, specifically if stripe api is ok to add to dev-packages

**Comment by mganisin** [comment]
I am not sold to the idea that such integration test should be here.
 - this represents another external dependency - doesn't have to be clear how to fullfil it
 - without proper setup this will fail always, if ran as of now, the result would be red (kind of a regression, but not a regression actually)

I am still undecided

**Comment by mganisin** [comment]
@averevki split it into two requests please, keep tests in separate PR, that will allow us to merge the feature and give bit more time to think about the tests

**Review by jsmolar — APPROVED** [review]

**Pull request merged**